exasol/error/_error.py

```diff
@@ -15,11 +17,11 @@ class Parameter:
 
 class Error:
     def __init__(
-        self,
-        code: str,
-        message: str,
-        mitigations: Union[str, Iterable[str]],
-        parameters: Dict[str, Union[str, Parameter]],
+            self,
+            code: str,
+            message: str,
+            mitigations: Union[str, Iterable[str]],
+            parameters: Dict[str, Union[str, Parameter]],
     ):
         # This function maybe flattened into or moved out of the constructor in the future.
```

Comment:
should this be a ticket, because with such comments, it usually doesn't happen

Reply:
I'll remove this comment.